app/states/texas/texas_downloader.py

```
@@ -7,90 +9,271 @@
+            df = (
+                df
+                .with_columns(
+                    [
+                        pl.col(col)
+                        .cast(pl.String) for col in df.columns
+                    ]
+                )
+            )
+
+            df.write_parquet(
+                file=cls.config.TEMP_FOLDER / f"{k}_{datetime.now():%Y%m%d}w.parquet",
```

Ruff (0.14.7)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)